src/app/page.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from 'next/image';
+import styles from './page.module.css';
+import { IMAGE_URL } from '@/constants';
+import { CategoryList } from '@/components/common';
+import RecipeRankingList from '@/components/rank/RecipeRankingList';
+import ProductRankingList from '@/components/rank/ProductRankingList';
+import ReviewRankingList from '@/components/rank/ReviewRankingList';
```

Comment:
이렇게 어떤가요? (저는 index.ts 만드는게 번거로워서 이렇게 안하긴 합니다..)

```suggestion
import { RecipeRankingList, ProductRankingList, ReviewRankingList } from '@/components/rank';
```